docs/csharp/programming-guide/concepts/attributes/index.md

```diff
@@ -1,171 +1,124 @@
-  
-```csharp  
-[DllImport("user32.dll")]  
-[DllImport("user32.dll", SetLastError=false, ExactSpelling=false)]  
-[DllImport("user32.dll", ExactSpelling=false, SetLastError=false)]  
-```  
-  
- The first parameter, the DLL name, is positional and always comes first; the others are named. In this case, both named parameters default to false, so they can be omitted. Refer to the individual attribute's documentation for information on default parameter values.  
-  
-### Attribute Targets  
- The *target* of an attribute is the entity to which the attribute applies. For example, an attribute may apply to a class, a particular method, or an entire assembly. By default, an attribute applies to the element that it precedes. But you can also explicitly identify, for example, whether an attribute is applied to a method, or to its parameter, or to its return value.  
-  
- To explicitly identify an attribute target, use the following syntax:  
-  
-```csharp  
-[target : attribute-list]  
-```  
-  
- The list of possible `target` values is shown in the following table.  
-  
-|Target value|Applies to|  
-|------------------|----------------|  
-|`assembly`|Entire assembly|  
-|`module`|Current assembly module|  
-|`field`|Field in a class or a struct|  
-|`event`|Event|  
-|`method`|Method or `get` and `set` property accessors|  
-|`param`|Method parameters or `set` property accessor parameters|  
-|`property`|Property|  
-|`return`|Return value of a method, property indexer, or `get` property accessor|  
-|`type`|Struct, class, interface, enum, or delegate|  
-  
- The following example shows how to apply attributes to assemblies and modules. For more information, see [Common Attributes (C#)](../../../../csharp/programming-guide/concepts/attributes/common-attributes.md).  
-  
-```csharp  
-using System;  
-using System.Reflection;  
-[assembly: AssemblyTitleAttribute("Production assembly 4")]  
-[module: CLSCompliant(true)]  
-```  
-  
- The following example shows how to apply attributes to methods, method parameters, and method return values in C#.  
-  
-```csharp  
-// default: applies to method  
-[SomeAttr]  
-int Method1() { return 0; }  
-  
-// applies to method  
-[method: SomeAttr]  
-int Method2() { return 0; }  
-  
-// applies to return value  
-[return: SomeAttr]  
-int Method3() { return 0; }  
-```  
-  
+> By convention, all attribute names end with the word "Attribute" to distinguish them from other items in the .NET Framework. However, you do not need to specify the attribute suffix when using attributes in code. For example, `[DllImport]` is equivalent to `[DllImportAttribute]`, but `DllImportAttribute` is the attribute's actual name in the .NET Framework.
```

Comment:
perhaps in the .NET Framework -> in the .NET Framework Class Library, or "of the .NET Framework type"

Reply:
If we rephrase it, then *Framework* might be omitted at all?